.drone.yml

```diff
@@ -0,0 +1,11 @@
+image: node0.10
```

Comment:
:+1: right now the way to do this is to `nvm install` node 5, since there isn't a node 5 image. E.g., https://github.com/Clever/district-advocacy/blob/master/.drone.yml#L27